modules/tools/AppFacCLI/out/session.txt

```diff
@@ -0,0 +1 @@
+{"UserName":"dilhasha.uom.gmail.com@uomcse","Cookie":"JSESSIONID=B777FA9788D79B9509CB76FD2AF60EE9"}
```

Comment:
No we don't need this. This is the session file generated by the tool. I removed it.